ddprof-test/src/test/java/com/datadoghq/profiler/jfr/CpuDumpSmokeTest.java

```diff
@@ -24,7 +24,9 @@ protected String getProfilerCommand() {
     @TestTemplate
     @ValueSource(strings = {"vm", "vmx", "fp", "dwarf"})
     public void test(@CStack String cstack) throws ExecutionException, InterruptedException, Exception {
-        test();
+        if (!isOnBlackList(cstack)) {
```

Comment:
This one is already guarding against disallowed stackwalkers - it is a templated test using `@CStackInjector` and that one will inject only compatible stackwalkers for testing (https://github.com/DataDog/java-profiler/blob/8070f3c9dd021329390d82a01eee23c6878c7df8/ddprof-test/src/test/java/com/datadoghq/profiler/junit/CStackInjector.java#L60)